<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryScreen</class>
 <widget class="QWidget" name="HistoryScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{font-family: Tamworth Gothic}
QLabel{color: #42b4ff; }</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/History.png</pixmap>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>330</y>
     <width>901</width>
     <height>381</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Tamworth Gothic</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(255,255,255,0);
border: none;
font-size: 25px;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>381</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_upArrowButton" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>108</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="downArrowLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>701</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/My Workouts/images/Down Arrow.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="upArrowLabel">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/My Workouts/images/Up-Arrow.png</pixmap>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_downArrowButton" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>690</y>
     <width>108</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_6" native="true">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>40</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="workoutsLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tamworth Gothic</family>
     <pointsize>31</pointsize>
    </font>
   </property>
   <property name="text">
    <string>17</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tamworth Gothic</family>
     <pointsize>31</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>workouts</string>
   </property>
  </widget>
  <widget class="QLabel" name="averageTimeLabel">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>170</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tamworth Gothic</family>
     <pointsize>31</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>28:05</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="averageCaloriesLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tamworth Gothic</family>
     <pointsize>31</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>522</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="averageDistanceLabel">
   <property name="geometry">
    <rect>
     <x>834</x>
     <y>170</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tamworth Gothic</family>
     <pointsize>31</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>4.8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="dateHeader">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>232</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Date Title (descending).png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="workoutHeader">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>220</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Workout Title (default).png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="timeHeader">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>220</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Time Title (default).png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="caloriesHeader">
   <property name="geometry">
    <rect>
     <x>666</x>
     <y>220</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Calories Title (default).png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="distanceHeader">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>220</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Distance Title (default).png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_date" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_workout" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_time" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_calories" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>230</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_distance" native="true">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>230</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InvisibleButton</class>
   <extends>QWidget</extends>
   <header>invisiblebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
